reports.html

```diff
@@ -2,26 +2,23 @@
 title: coala GSoC project reports
 ---
 
-<h1>{{ page.title }}</h1>
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>{{ page.title }}</title>
+        <link rel="stylesheet" href="../resources/css/style.css">
+    </head>
 
-<h2>2018</h2>
-
-<ul>
-  {% for report in site.reports %}
-    {% capture year %}{{ report.date | date: '%Y' }}{% endcapture %}
-    {% if year == '2018' %}
-      <li><a href="{{ report.url }}.html">{{ report.student }}</a>
-    {% endif %}
-  {% endfor %}
-</ul>
-
-<h2>2017</h2>
-
-<ul>
-  {% for report in site.reports %}
-    {% capture year %}{{ report.date | date: '%Y' }}{% endcapture %}
-    {% if year == '2017' %}
-      <li><a href="{{ report.url }}.html">{{ report.student }}</a>
-    {% endif %}
-  {% endfor %}
-</ul>
+    <body class='report'>
+        <h1>{{ page.title }}</h1>
+        {% assign reportsByYear = site.reports | group_by_exp:"post", "post.date | date: '%Y'" %}
```

Comment:
Shouldn't it be `group_by_exp:"report"` and `report.date`?

Reply:
@srivama Fixed